config/config_helpers_test.go

```diff
@@ -624,8 +637,8 @@ TERRAGRUNT_HIT","")}/bar`,
 			nil,
 			terragruntOptionsForTestWithEnv(t, "/root/child/"+DefaultTerragruntConfigPath, map[string]string{"TEST_ENV_TERRAGRUNT_OTHER": "SOMETHING"}),
 			"",
-			InvalidInterpolationSyntax(`${get_env("TEST_ENV_
-TERRAGRUNT_HIT","")}`),
+			InvalidGetEnvParams(`"TEST_ENV_
```

Comment:
Here, the newline would previously fail to even recognize as valid interpolation syntax.

Since the new parser actually handles newlines, the error that gets thrown is actually `InvalidGetEnvParams`